packages/@aws-cdk/aws-apigateway/test/test.lambda-api.ts

```diff
@@ -126,9 +126,12 @@ export = {
               },
               ":lambda:path/2015-03-31/functions/",
               {
-                "Ref": "alias68BF17F5"
+                "Fn::GetAtt": [
+                  "handlerE1533BD5",
+                  "Arn"
+                ]
               },
-              "/invocations"
+              ":my-alias/invocations"
```

Comment:
The fact that we're changing this worries me a bit as well... maybe consult with somebody that knows a lot about the API Gateway construct whether these changes are OK? I don't feel competent enough in that area to comment on it.